examples/chat.rs

```diff
@@ -297,15 +296,18 @@ impl Ticket {
 /// Serializes to base32.
 impl fmt::Display for Ticket {
```

Comment:
Ticket has serialize and deserialize, which as far as I can tell still uses base32. Why not use that instead of doing it manually here?